R/sun-methods.R

```diff
@@ -120,6 +175,8 @@ setMethod("sunriset", signature(crds="matrix", dateTime="POSIXct"),
                        direction=direction, POSIXct.out=POSIXct.out)
           })
 
+#' @description Method for SpatialPoints objects.
```

Comment:
these description lines can be omitted, since they lead to an unwanted line `Method for SpatialPoints objects.` in the main function documentation